roles/nginxplus/files/conf/http/api.conf

```diff
@@ -0,0 +1,20 @@
+#
```

Comment:
@kayiwa if this file needed?

Reply:
every time we tunnel to the LB and need to add or remove services using the web UI we need this file.